### DISCUSSION TITLE
连接点不稳定，无法使用请求退款

### DISCUSSION THREAD

**Discussion by liend123** [getlantern/lantern-forum-cn#998] [Q&A]
![Screenshot_20241026_123013_com eg android AlipayGphone](https://github.com/user-attachments/assets/037c8e2d-f024-405e-9550-d421cd320815)

**Comment by liend123** [comment]
我的邮箱是liend123@163.com

**Comment by MaxMKMax** [comment]
@liend123 您好. 已经帮您提交上去了
